css/style.css

```diff
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@300&display=swap');
```

Comment:
This should all be indented to 2 spaces, or the rest of the project should be indented to 4, which you prefer is up to you but it should be consistent across all CSS, JS and HTML files